Makefile

```diff
@@ -3,7 +3,7 @@ GOOS = $(shell go env GOOS)
 GOARCH = $(shell go env GOARCH)
 
 GO_BIN := $(shell pwd)/.bin
-OVERRIDE_GOCI_LINT_V := v1.55.2
+OVERRIDE_GOCI_LINT_V := v1.59.1
```

Comment:
Needed this because I'm using a 1.22-specific feature that the older lint didn't understand (range over int)